app-accessibility/pipertts/pipertts-1.0.recipe

```diff
@@ -0,0 +1,63 @@
+SUMMARY="Piper TTS is an offline text-to-speech program using the NCNN library"
+DESCRIPTION="Piper TTS is an offline text-to-speech program using the NCNN library \
+to provide natural, Human-like voices. Different voice models can be downloaded from \
+https://huggingface.co/gyroing/PiperTTS-NCNN-Models/tree/main."
+HOMEPAGE="https://github.com/gyroing/PiperTTS-HaikuOS/"
+COPYRIGHT="2025 GYROING"
+LICENSE="BSD (3-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/gyroing/PiperTTS-HaikuOS/archive/refs/tags/piper.tar.gz"
+CHECKSUM_SHA256="7d391de5f9664d0e05e357bb206ec827dc5c0b6cee3002e03134d778dc37066f"
+srcGitRev="d7df277f9cc15a00b19877ab75b287ade42a22ab"
+SOURCE_URI_2="https://github.com/Tencent/ncnn/archive/${srcGitRev}.tar.gz"
+CHECKSUM_SHA256_2="989bb0f7b8a00c24b3b297cdc7f23c6932a5676731c7dda24ea29dfc8eae4668"
+SOURCE_DIR="PiperTTS-HaikuOS-piper"
+
+ARCHITECTURES="x86_64"
+
+PROVIDES="
+	pipertts = $portVersion
+	cmd:pipertts = $portVersion
+	"
+
+REQUIRES="
+	haiku
+	cmd:espeak
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	cmd:cmake
+	cmd:gcc
+	cmd:make
```

Comment:
These 3 should go to BUILD_PREREQUIRES (no new line between BUILD_REQUIRES/BUILD_PREREQUIRES).